cmake/options.cmake

```diff
@@ -139,5 +139,10 @@ if (DEFINED RPMSG_BUFFER_SIZE)
 endif (DEFINED RPMSG_BUFFER_SIZE)
 
 option (WITH_DOC "Build with documentation" OFF)
+option (WITH_RPMSG_TRACE "Enable Rpmsg Device Trace" OFF)
+
+if (WITH_RPMSG_TRACE)
```

Comment:
Please explain commit mesasge in detail what this option is for.

Reply:
Adding this is to track the application and release of rpmsg buffer

Reply:
Have you checked virtqueue_dump functionality? Is it not enough for above purpose?

Reply:
virtqueue_dump is not enough for record historical information of buffer application and release